compiler/src/iree/compiler/API/Internal/IREEGPUDialectCAPI.cpp

```diff
@@ -213,3 +213,61 @@ MlirAttribute ireeGPULoweringConfigAttrGetAttributes(MlirAttribute attr) {
                   unwrap(attr))
                   .getAttributes());
 }
+
+ireeGPUTileSizes ireeGPULoweringConfigAttrGetTileSizes(MlirAttribute attr) {
+  assert(ireeAttributeIsAGPULoweringConfigAttr(attr));
+  ireeGPUTileSizes tilesizes = {};
+  auto loweringConfigAttr =
+      llvm::cast<mlir::iree_compiler::IREE::GPU::LoweringConfigAttr>(
+          unwrap(attr));
+
+  auto workgroups = loweringConfigAttr.getWorkgroupTileSizes();
+  tilesizes.workgroupTileSizes = workgroups.data();
+  tilesizes.numWorkgroupTileSizes = workgroups.size();
+
+  auto reduction = loweringConfigAttr.getStaticTilingLevelSizes(
+      static_cast<int64_t>(
+          mlir::iree_compiler::IREE::GPU::TilingLevel::Reduction),
+      nullptr);
+  tilesizes.reductionTileSizes = reduction.data();
+  tilesizes.numReductionTileSizes = reduction.size();
+
+  return tilesizes;
+}
+
+ireeGPUSubgroupCountInfo
+ireeGPULoweringConfigAttrGetSubgroupCount(MlirAttribute attr) {
+  assert(ireeAttributeIsAGPULoweringConfigAttr(attr));
+  mlir::DictionaryAttr dict =
+      llvm::cast<mlir::iree_compiler::IREE::GPU::LoweringConfigAttr>(
+          unwrap(attr))
+          .getAttributes();
+
+  constexpr mlir::StringLiteral kSubgroupMCountName = "subgroup_m_count";
+  constexpr mlir::StringLiteral kSubgroupNCountName = "subgroup_n_count";
+
+  mlir::IntegerAttr subgroup_m_count_attr =
+      dict.getAs<mlir::IntegerAttr>(kSubgroupMCountName);
+  mlir::IntegerAttr subgroup_n_count_attr =
+      dict.getAs<mlir::IntegerAttr>(kSubgroupNCountName);
```

Comment:
Don't we have helpers for this in the dialect headers or the attr interface?

Reply:
Here: https://github.com/iree-org/iree/blob/a1664e30a6a53850f689f32086a8b0c45bed327b/compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/GPULoweringConfigUtils.h#L22-L23